validator/client/beacon-api/beacon_block_json_helpers.go

```diff
@@ -155,3 +155,23 @@ func jsonifyAttestationData(attestationData *ethpb.AttestationData) *apimiddlewa
 		},
 	}
 }
+
+func fillByteSlice(sliceLength int, value byte) []byte {
```

Comment:
`fillByteSlice` and `fillByteArraySlice` are used only in tests and thus shouldn't be compiled into the binary. You can use a `testing` package, an example being `/consensus-types/blocks/testing` (note the `testonly = True` argument in bazel file)